api/v3/Attachment.php

```diff
@@ -281,7 +281,7 @@ function __civicrm_api3_attachment_find($params, $id, $file, $entityFile, $isTru
   }
 
   $select = CRM_Utils_SQL_Select::from('civicrm_file cf')
-    ->join('cef', 'INNER JOIN civicrm_entity_file cef ON cf.id = cef.file_id')
+    ->join('cef', 'LEFT JOIN civicrm_entity_file cef ON cf.id = cef.file_id')
```

Comment:
We want INNER though for regular non-custom attachments, because they _should_ have an entry in civicrm_entity_file. We don't want entity_table/entity_id being null in that case.